Source/WebInspectorUI/UserInterface/Views/ConsoleSession.js

```diff
@@ -46,7 +46,7 @@ WI.ConsoleSession = class ConsoleSession extends WI.Object
             headerText = WI.UIString("Page reloaded at %s");
             break;
         case WI.ConsoleSession.NewSessionReason.PageNavigated:
-            headerText = WI.UIString("Page navigated at %s");
+            headerText = WI.UIString("Navigated to %s").format(WI.networkManager.mainFrame.url);
```

Comment:
i think it's valuable to have the timestamp for when the navigation occurred, especially if the developer has enabled showing timestamps for console messages

but i also see a lot of value in showing the URL

what if this was `"Navigated to %s at %s"` so that it has both?

Reply:
yes agreed.